content/blog/deprecating-excalidraw-electron/index.md

```diff
@@ -0,0 +1,144 @@
+
+### Installable Progressive Web App
+
+Excalidraw today is an [installable](https://web.dev/installable/) Progressive Web App with a [service worker](https://excalidraw.com/service-worker.js) and a [Web App Manifest](https://excalidraw.com/manifest.json). It caches all its resources in two caches, one for fonts and font-related CSS, and one for everything else.
+
+<figure>
+  <img src="excalidraw-cache.png" alt="Chrome DevTools Application tab showing the two Excalidraw caches.">
+  <figcaption>Excalidraw's cache contents</figcaption>
+</figure>
+
+This means the application is fully offline-capable and can run without a network connection. Chromium-based browsers on both desktop and mobile prompt the user if they want to install the app. You can see the installation prompt in the screenshot below.
+
+<figure>
+  <img src="install-excalidraw.png" alt="Excalidraw prompting the user to install the app in Chrome on macOS.">
+  <figcaption>The Excalidraw install dialog in Chrome</figcaption>
+</figure>
+
+Excalidraw is configured to run as a standalone application, so when you install it, you get an app that runs in its own window. It is fully integrated in the operating system's multitasking UI and gets its own app icon on the home screen, Dock, or task bar; depending on what platform you install it.
+
+<figure>
+  <img src="excalidraw-pwa.png" alt="Excalidraw running in its own window.">
+  <figcaption>The Excalidraw PWA in a standalone window</figcaption>
+</figure>
+
+<figure>
+  <img src="excalidraw-icon.png" alt="Excalidraw icon on the macOS Dock.">
+  <figcaption>The Excalidraw icon on the macOS Dock</figcaption>
+</figure>
+
+### File system access
+
+Excalidraw makes use of [browser-nativefs](https://github.com/GoogleChromeLabs/browser-nativefs) for accessing the file system of the operating system. On supporting browsers, this allows for a true open-->edit-->save workflow and actual over-saving and "save as", with a transparent fallback for other browsers. You can learn more about this feature in my blog post [Reading and writing files and directories with the browser-nativefs library](browser-nativefs/).
+
+### Drag and drop support
+
+Files can be dragged and dropped onto the Excalidraw window just as in native applications. On a browser that supports the [File System Access API](https://web.dev/file-system-access/), a dropped file can be immediately edited and the modifications be saved in the original file. This is so intuitive that you sometimes forget that you are dealing with a Web app.
```

Comment:
```suggestion
Files can be dragged and dropped onto the Excalidraw window just as in native applications. On a browser that supports the [File System Access API](https://web.dev/file-system-access/), a dropped file can be immediately edited and the modifications be saved to the original file. This is so intuitive that you sometimes forget that you are dealing with a Web app.
```

Reply:
Btw, should we capitalize `Web app` when not talking about PWA?

Reply:
I'm in the "the Web" and "the Internet" character casing camp.

Reply:
Dealer's choice :)